va/validation-authority.go

```diff
@@ -425,7 +426,7 @@ func (va *ValidationAuthorityImpl) validateDNS01(identifier core.AcmeIdentifier,
 	// Compute the digest of the key authorization file
 	h := sha256.New()
 	h.Write([]byte(challenge.KeyAuthorization.String()))
-	authorizedKeysDigest := hex.EncodeToString(h.Sum(nil))
+	authorizedKeysDigest := base64.RawURLEncoding.EncodeToString(h.Sum(nil))
```

Comment:
`base64.RawURLEncoding` omits the pad character. This should use `base64.URLEncoding` to be really compliant with the acme spec.